unsubscribe-automation/src/main.ts

```diff
@@ -0,0 +1,199 @@
+import { chromium, Page } from "playwright";
+import { GoogleGenerativeAI } from "@google/generative-ai";
+import { z } from "zod";
+
+import dotenv from "dotenv";
+
+dotenv.config();
+
+const genAI = new GoogleGenerativeAI(process.env.GOOGLE_AI_API_KEY || "");
```

Comment:
Use vercel `ai` as this is what we use in the core app too.

Reply:
https://sdk.vercel.ai/docs/ai-sdk-core/overview

Reply:
Added google ai using vercel ai sdk